LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 FRENDS
```

Comment:
Please update the license. See: https://github.com/FrendsPlatform/Frends.AzureEventHub/blob/main/LICENSE#L3